e2e/utils.go

```diff
@@ -574,6 +574,23 @@ func getRBDImageSpec(pvcNamespace, pvcName string, f *framework.Framework) (stri
 	return fmt.Sprintf("replicapool/csi-vol-%s", imageID), nil
 }
 
+func getCephFSVolumeName(pvcNamespace, pvcName string, f *framework.Framework) (string, error) {
```

Comment:
Function should be rewritten to re-use common parts of the above function (getRBDImageSpec). Will do in a subsequent revision.